methods/guest/Cargo.toml

```diff
@@ -7,11 +7,11 @@ edition = "2021"
 
 [target.riscv32im-unknown-none-elf.dependencies]
```

Comment:
I think the target triple changed, so might need to adjust this to:
```
[target.riscv32im-risc0-zkvm-elf.dependencies]
```

Reply:
It did change (good catch!) I noticed it changed because it couldn't find the crates. I fixed it by changing the header to `[dependencies]`, which (delightfully!) works just fine with `risc0-build`. See https://github.com/risc0/risc0-rust-starter/blob/main/methods/guest/Cargo.toml